core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -470,15 +447,6 @@
    */
   @CarbonProperty
   public static final String CARBON_DATE_FORMAT = "carbon.date.format";
-  /**
-   * STORE_LOCATION_HDFS
-   */
-  @CarbonProperty
-  public static final String STORE_LOCATION_HDFS = "carbon.storelocation.hdfs";
```

Comment:
can you please explain, why need to remove :  "STORE_LOCATION_HDFS" ?

Reply:
it is not being used